src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,26 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService)
+                injector.getInstance(DriverService.class);
+        Driver driver = new Driver();
+        driver.setName("Lili");
+        driver.setLicenseNumber("mn2345");
+        driverService.create(driver);
+        driverService.update(new Driver(6L, "Liliya", "nn8900"));
+        driverService.delete(5L);
```

Comment:
who is the 5th driver?
are you sure if I clone your project, this code will work correctly?

Reply:
I agree it's better to use id of drivers you've created